packages/manager/modules/overthebox/src/overthebox/migration/resume/overTheBox-migration-resume.controller.js

```diff
@@ -0,0 +1,106 @@
+import {
+  NO_HARDWARE,
+  HELP_BILLING_URL,
+} from '../overTheBox-migration.constant';
+
+export default class OverTheBoxMigrationContactCtrl {
+  /* @ngInject */
+  constructor($scope, $translate, OverTheBoxMigrationResumeService, TucToast) {
+    this.$scope = $scope;
+    this.$translate = $translate;
+    this.OverTheBoxMigrationResumeService = OverTheBoxMigrationResumeService;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.loading = true;
+    this.NO_HARDWARE = NO_HARDWARE;
+    this.HELP_BILLING_URL = HELP_BILLING_URL;
+
+    if (this.offer.selectedHardware !== NO_HARDWARE) {
+      this.hardwarePrice = {
+        value: '0',
+        currencyCode: 'EUR',
+        text: '0.00 €',
+      };
+    }
+
+    this.getFirstMonthPrice();
+
+    this.getOfferTargetDetail();
+  }
+
+  getFirstMonthPrice() {
+    if (this.offer.selectedHardware !== NO_HARDWARE) {
+      const value =
+        parseFloat(this.offer.price.value) +
+        parseFloat(this.offer.hardwarePrice.value);
+      this.firstMensuality = {
+        value,
+        currencyCode: 'EUR',
+        text: `${parseFloat(value).toFixed(2)} €`,
+      };
+    }
+  }
+
+  getOfferTargetDetail() {
+    const isSelectedHardware = this.offer.selectedHardware !== NO_HARDWARE;
+    return this.OverTheBoxMigrationResumeService.getOfferTargetDetail(
+      this.serviceName,
+      this.offer.offer,
+      isSelectedHardware,
+      this.contact?.address.id,
+    )
+      .then(({ data }) => {
+        this.contracts = data.contracts;
+      })
+      .catch((error) => {
+        // Display error message
+        const msg = error?.data?.message;
+        this.TucToast.error(
+          this.$translate.instant(
+            'overthebox_migration_resume_get_offer_detail_error',
+            {
+              errorMsg: msg,
+            },
+          ),
+        );
+      })
+      .finally(() => {
+        this.loading = false;
+      });
+  }
+
+  launchMigration() {
+    this.loading = true;
+    const selectedHardware =
+      this.offer.selectedHardware !== NO_HARDWARE
+        ? this.offer.selectedHardware
+        : '';
+    return this.OverTheBoxMigrationResumeService.migrateOffer(
+      this.serviceName,
+      this.offer.offer,
+      selectedHardware,
+      this.contact?.address.id,
+    )
+      .then(({ data }) => {
+        this.isMigrationOK = true;
+        this.commandOrder = data;
+      })
+      .catch((error) => {
+        // Display error message
+        const msg = error?.data?.message;
+        this.TucToast.error(
+          this.$translate.instant(
+            'overthebox_migration_resume_launch_migration_error',
+            {
+              errorMsg: msg,
+            },
+          ),
+        );
```

Comment:
```suggestion
        this.TucToast.error(
          this.$translate.instant(
            'overthebox_migration_resume_launch_migration_error',
            {
              errorMsg: error?.data?.message,
            },
          ),
        );
```

Reply:
No it doesn't work, it generated an errror with the $translate, that's why I did this.